tests/testapp/testapp/grains/rich_text.py

```diff
@@ -47,6 +47,13 @@
     'source': 'DocumentSource',
     'decorator': 'Document',
 }
+DRAFT_ENTITY_TYPE_BUTTON = {
+    'label': 'Button',
+    'type': 'BUTTON',
```

Comment:
Pythonic or not, using strings here can indeed be considered bad practice so I'll change that.